CHANGES.md

```diff
@@ -2,9 +2,13 @@
 
 ## 0.12.0 (yyyy-mm-dd)
 
+### Improvements
+
+-   Return JSON fields as dicts in `read_dataframe` (#556).
```

Comment:
```suggestion
-   Return JSON fields (as identified by GDAL) as dicts in `read_dataframe` and listing of GDAL data types and subtypes to `read_info` (#556).
```

Should this get marked as a breaking change, to make it more obvious?  Though usage of this is likely rare, if folks already had code that was parsing the JSON strings to dictionaries, this update would break that code.

Reply:
True: breaking change.